### DISCUSSION TITLE
Change editor rate adjust to use time stretching

### DISCUSSION THREAD

**Discussion by durshlakk** [ppy/osu#25024] [Ideas]
Hello, dear developers. I want to suggest working on slowing down in the osu!lazer editor. In osu!stable, if you slow down the music, then the sounds of the beat and music are very well heard there, and in osu!lazer, especially by 25-50%, almost no damn thing is heard. Can you fix it? This makes it inconvenient to work on the map

**Comment by NafeeJ** [comment]
are you talking about [this](https://github.com/ppy/osu/discussions/23789)?

**Reply by peppy** [reply]
Yes, that's the same issue at discussion.
